packages/serve/package.json

```diff
@@ -13,24 +13,18 @@
   "dependencies": {
     "@webpack-cli/utils": "^1.0.1-alpha.5",
     "chalk": "3.0.0",
-    "cross-spawn": "6.0.5",
-    "inquirer": "6.5.1",
-    "webpack-cli": "^4.0.0-beta.8",
-    "webpack-dev-server": "^3.9.0"
+    "cross-spawn": "6.0.5"
```

Comment:
Hm, where we use `cross-spawn` in out code? Maybe it should be dev dependencies?

Reply:
The only place I see it used in the repo is part of the `package-utils` library